openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/TargetPacketItemChooser.vue

```diff
@@ -21,16 +21,15 @@
 -->
 
 <template>
-  <!-- tgt-pkt-item-chooser class used by Graph.vue to size the graph -->
-  <div class="pt-4 tgt-pkt-item-chooser">
+  <div class="pt-4 pb-4">
     <v-row>
       <v-col :cols="colSize" class="select" data-test="select-target">
         <v-autocomplete
           label="Select Target"
           hide-details
           density="compact"
           variant="outlined"
-          @update:model-value="targetNameChanged"
+          @update:modelValue="targetNameChanged"
```

Comment:
Was `model-value` not working? Vue _should_ handle the camel/kebab conversion, and kebab is the convention for templates